src/main/java/org/tarantool/jdbc/ds/SQLDataSource.java

```diff
@@ -0,0 +1,163 @@
+package org.tarantool.jdbc.ds;
+
+import org.tarantool.jdbc.SQLConnection;
+import org.tarantool.jdbc.SQLProperty;
+
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.sql.SQLNonTransientException;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+import javax.sql.DataSource;
+
+/**
+ * Simple {@code java.sql.DataSource} implementation.
+ */
+public class SQLDataSource implements TarantoolDataSource, DataSource {
+
+    private PrintWriter logWriter;
+    private String description;
```

Comment:
I see pgjdbc allow only to get a description of a source (and provides meaningful string here). Should we too?